### DISCUSSION TITLE
Apple AirTags

### DISCUSSION THREAD

**Discussion by LordNex** [ESPresense/ESPresense#150] [Ideas]
I was wondering if there was going to be support for Apple AirTags as well? We use them here as we are all in an Apple eco system with our mobile devices anyway so it makes tracking things like my keys and wallet real easy and shareable with family in FindMy. I know they kind of integrate through the iCloud HA Integration but i havent been able to track them as far as location

**Comment by corgan2222** [comment]
>I know they kind of integrate through the iCloud HA Integration but i havent been able to track them as far as location

Really? The time I tested it, there has been no useable solution at that time. So can you see your Airtag in HA with the iCloud integration?
The only place I could see my Airtag was in the "where is" App.
Do Apple support more now?

**Reply by LordNex** [reply]
Sorry for the late reply. Been pretty busy as of late. 

Yea, once you pull it in you basically get all the devices on your account. I also have Family mode turned on so I see all my family's devices as well. Haven't found a really good use for it yet but I have an airtag on my wallet and one on my keys. I'm thinking about getting some with collars for our pets as well. The reason I like AirTags is if they are lost and get close to any idevice, that device will anonymously notify you that it's been located and give you a general area. Once there you just follow the arrow to your device.

**Reply by TheNuttyP** [reply]
Sadly AirTags do not show up in Family mode. They get associated with a single phone and that's the only one that can see them, even with Family enabled.

We have 6 of them on various things in our family, but I can only see 'mine' in the Find My app.

**Reply by LordNex** [reply]
They must have taken that out because I remember commenting about it to my with because I'm constantly asking her where shit is. Strange, I'm using icloud3 which lets me check any device, minus AirTags, like each persons phone and watch. Then I passively scan the iPads but those are not as portable

**Comment by LordNex** [comment]


**Reply by TheNuttyP** [reply]
We do. We have the full Apple One plan with 2TB with Apple Family sharing. I can see the tags I set up on my phone, but none of us can see each other's tags. I'd love to get it working the way you're describing.

Are you using a single Apple ID on all your family's devices, or does everyone have their own Apple ID?

**Comment by LordNex** [comment]


**Reply by TheNuttyP** [reply]
Totally understood. Sounds like your setup is pretty much similar to mine. We wanted to do the same thing and all be able to see each other's tags. I hope they (Apple) add that feature (add it back?) in a future update.

**Reply by LordNex** [reply]
Agreed
